.github/workflows/ci-dependabot-fixup.yml

```diff
@@ -1,118 +0,0 @@
-        then
-            echo "This PR was not created by Dependabot. No further action is being taken."
-            echo "skip=true" >> $GITHUB_OUTPUT
-            exit 0;
-        fi
-
-        # Ensure only Dependabot commits.
-        git fetch --no-tags origin +refs/heads/${BASE_BRANCH}:refs/remotes/origin/${BASE_BRANCH}
-        if git log origin/${BASE_BRANCH}..HEAD --pretty=format:'%an' | grep -v '^dependabot\[bot\]$' | grep -q .
-        then
-            echo "This PR has commits not by Dependabot."
-            echo "skip=true" >> $GITHUB_OUTPUT
-            exit 0;
-        fi
-
-        # Ensure Go dependency updates.
-        if ! git diff --name-only origin/${BASE_BRANCH}..HEAD | grep -q 'go\.mod$'
-        then
-          echo "This PR does not update any Go dependencies."
-          echo "skip=true" >> $GITHUB_OUTPUT
-          exit 0;
-        fi
-
-        echo "All commits are by Dependabot and update Go modules."
-      env:
-          BASE_BRANCH: ${{ github.base_ref }}
-
-    - name: Set up Go
-      if: steps.check.outputs.skip != 'true'
-      uses: actions/setup-go@v5
-      with:
-        go-version: "1.22.x"
-
-    - name: Build gomod updater
-      if: steps.check.outputs.skip != 'true'
-      working-directory: tools/gomod-updater
-      run: go build
-
-    # Dependabot titles are:
-    #      <prefix>: bump <dependancy name> from <frmo_version> to <to_version> in <path>
-    # e.g. client-sdk/go: bump github.com/ethereum/go-ethereum from 1.12.1 to 1.13.3 in /client-sdk/go
-    # as long as the (configurable) <prefix> is without whitespace, the bellow parsing should work.
-    - name: Try extracting package name and version
-      if: steps.check.outputs.skip != 'true'
-      id: extract
-      run: |
-        title="${{ github.event.pull_request.title }}"
-        repo=$(echo $title | awk '{print $3}')
-        version=$(echo $title | awk '{print $7}')
-
-        # Set the output variables for subsequent steps
-        echo "repo=$repo" >> $GITHUB_OUTPUT
-        echo "version=$version" >> $GITHUB_OUTPUT
-
-    - name: Run gomod updater
-      if: steps.check.outputs.skip != 'true'
-      run: |
-        # Ensure client-sdk is updated first, as some other packages depend on it.
-        client_sdk="./client-sdk/go/go.mod"
-        file_list=$(echo $client_sdk | awk -vORS=, '{ print $1 }')
-        file_list+=$(find . -type f -name 'go.mod' ! -path "$client_sdk" | awk -vORS=, '{ print $1 }' | sed 's/,$/\n/')
-        tools/gomod-updater/gomod-updater ${{ steps.extract.outputs.repo }} ${{ steps.extract.outputs.version }} --packages "$file_list"
-
-    - name: Commit and push all changed files
-      if: steps.check.outputs.skip != 'true'
-      env:
-        CI_COMMIT_MESSAGE: Dependabot dependencies fixup 👷
-        CI_COMMIT_AUTHOR: oasisprotocol-bot
-      run: |
-        git config --global user.name "${{ env.CI_COMMIT_AUTHOR }}"
```

Comment:
This was a bit more risky for injections than `$CI_COMMIT_AUTHOR`, but it's a constant `CI_COMMIT_AUTHOR: oasisprotocol-bot`